test/test.coffee

```diff
@@ -462,13 +487,20 @@ describe "StoreIt!", ->
                 @forEach.should.have.been.calledThrice
 
         describe "when calling clear", ->
-            beforeEach ->
+            beforeEach (done) ->
                 @publishReady = sinon.stub()
                 service.on("ready", @publishReady)
-                service.set("testkey1", 1)
-                service.set("testkey2", 2)
-                service.set("testkey3", 3)
-                service.clear()
+                @result1 = service.set("testkey1", 1)
+                @result2 = service.set("testkey2", 2)
+                @result3 = service.set("testkey3", 3)
+                @result1.promise.then(
+                    () => @result2.promise.then(
+                        () => @result3.promise.then(
+                            () =>
```

Comment:
Line contains inconsistent indentation<br>Unnecessary fat arrow<br>Empty parameter list is forbidden